### ISSUE TITLE
Incorrect methods being called

### ISSUE THREAD

**Issue by sevenpixels** [gtm-templates-simo-ahava/ip-geolocation-api#2]
All of the methods need to be updated, remove `Parameter` from them.

Currently not working
```
  if (data.customIp) {
    ipGeo.setIPAddressParameter(data.customIp);
  }
  if (data.responseLanguage !== 'en') {
    ipGeo.setLanguageParameter(data.responseLanguage);
  }
  if (data.fieldsToInclude) {
    ipGeo.setFieldsParameter(data.fieldsToInclude);
  }
  if (data.fieldsToExclude) {
    ipGeo.setExcludesParameter(data.fieldsToExclude);
  }
```

Working
```
  if (data.customIp) {
    ipGeo.setIPAddress(data.customIp);
  }
  if (data.responseLanguage !== 'en') {
    ipGeo.setLanguage(data.responseLanguage);
  }
  if (data.fieldsToInclude) {
    ipGeo.setFields(data.fieldsToInclude);
  }
  if (data.fieldsToExclude) {
    ipGeo.setExcludes(data.fieldsToExclude);
  }
```

**Comment by davilayang** [comment]
Also this will break the Tag if values are given in the "Other Settings => Fields To Include or Fields to Exclude". I.e. setting fields doesn't work with current deployment.

**Comment by sahava** [comment]
Fixed in 9ecb8961c1d37d92216a0f827b5be901027537ab
